website/pages/docs/connect/connect-internals.mdx

```diff
@@ -57,14 +59,22 @@ Connect endpoints](/api/agent/connect) implement local caching, background
 updating, and support blocking queries. Most API calls operate on purely local
 in-memory data.
 
+## Intention Authorization
```

Comment:
This seems more appropriate to list on https://www.consul.io/docs/connect/proxies/integrate.html.

Reply:
Moved and rearranged quite a lot.